src/components/Toggle/Toggle.scss

```diff
@@ -93,15 +95,15 @@
       }
     }
     & .l3-style-toggle_text {
-      font-size: 12px;
-      line-height: 16px;
+      font-size: $spacing-small-medium;
+      line-height: $spacing-medium;
     }
   }
 
   &_kind_secondary {
     & .l3-style-toggle_toggle {
       &--not-selected {
-        background: rgba(255, 255, 255, 0.2);
+        background: $white-2;
```

Comment:
same here , please in the mixin create for example variable  toggle-primary-bg-color: $black-2 and then use with them-prop